### ISSUE TITLE
Fuel resource slider duplication in tooltip window while in editor

### ISSUE THREAD

**Issue by icedown** [sswelm/KSP-Interstellar-Extended#131]
I am using KSP 1.3.1 with KSPIE 1.16.0

This is a visual bug, it does not affect gameplay.

Reproduction:
* Open VAB
* Select Molten Salt Reactor
* Right click on it to bring up tool tip
* Click "Swap Fuel"

I have confirmed this happens with the "Candle" and "Solid Core Nuclear engine" but does not affect the "TORY".  

If you hide the tooltip and reopen it, the correct fuel displays with no problems and when swapping, the correct resource  shows up in other mods at the correct levels.

[Output Log](https://pastebin.com/TWjpa5Y1)
[Screen Shots](https://imgur.com/VFV4rVQ)

**Comment by sswelm** [comment]
I willl fix it then overhauling whole breeding mechanic

**Comment by arivaldh** [comment]
I'll look into the issue. It's also present when EVAing and pressing and DOES affect gameplay (resource is magically created).

**Comment by arivaldh** [comment]
https://github.com/arivaldh/KSP-Interstellar-Extended/commit/66856b872f07e4f1d1e557aafee24efea12e819c

 Fixed Swap Fuel in MSRGC

- Swap Fuel from Editor NOT executed in Flight
- Missing Resource tanks generated in OnStart
- Switch Fuel tank function, that does NOT switch between modes.
- Fuel tank in Reactor not used to Unfuel/Fuel as an external fuel source
- Swap Fuel functions disable/enable proper internal fuel tanks.

Fixed all issues that I have found in Molten Salt Fission Reactor.
